lib/logger.js

```diff
@@ -1,23 +1,25 @@
-var winston = require('winston');
+const winston = require('winston');
 
+const logger = new winston.Logger({});
 
-var logger = new (winston.Logger)({});
-
-// Log into the console
 logger.add(winston.transports.Console, {
 	colorize: true,
-  level: (process.env.NODE_ENV === 'development') ? 'info' : 'warning'
+	level:
+    process.env.NODE_ENV === 'development' ||
+      process.env.ENABLE_TN_LOGS === 'true'
```

Comment:
wondering if theres any point shortening it? ENABLE_THREADNEEDLE_LOGS would be a lot more clear